scripts/e2e.sh

```diff
@@ -1,59 +1,132 @@
 #!/bin/bash
 
+readonly usage="Usage: e2e.sh -u <docker-username> -p <docker-password> --cluster-url <url> --cluster-token <token> --registry-name <name> --registry-namespace <namespace>"
 readonly SERVICE_ACCOUNT="travis-tests"
 
-# login_cluster : Download oc cli and use it to log into our persistent cluster
-login_cluster(){
+# setup_env: Download oc cli, log into our persistent cluster, and create a test project
+setup_env() {
+    echo "****** Installing OC CLI..."
     # Install kubectl and oc
     curl -L https://mirror.openshift.com/pub/openshift-v4/clients/ocp/stable-4.3/openshift-client-linux.tar.gz | tar xvz
```

Comment:
We should update to at least 4.6.0 version (long supported version): https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.6.0/openshift-client-linux.tar.gz